core/store/migrate/migrations/0193_functions_alter_version_type.sql

```diff
@@ -0,0 +1,7 @@
+-- +goose Up
+
+ALTER TABLE "s4".shared ALTER COLUMN version TYPE BIGINT;
```

Comment:
Can you add (or modify) a test to actually save a version larger than INT?

Reply:
Added.